### ISSUE TITLE
RSS link is not the same as atom rel=self

### ISSUE THREAD

**Issue by nivekuil** [yogthos/clj-rss#22]
The link field in RSS is the link of the website, but this library also uses it to set this atom attribute https://github.com/yogthos/clj-rss/blob/master/src/clj_rss/core.clj#L137 which is supposed to be the URL of the feed itself

**Comment by yogthos** [comment]
I see, sounds like adding an additional `:url` key would be the way to go then?

**Comment by nivekuil** [comment]
would work,' or maybe `:feed-url` to be less ambiguous

**Comment by yogthos** [comment]
Good call, I've added the feature in [this](https://github.com/yogthos/clj-rss/commit/934e92646b6b263c9a18af02f185d112fefe3cb0) commit. Let me know if that looks good on your end and I can push out a new version. I've kept the original behavior when `:feed-url` is not specified for backwards compatibility.
